src/main/kotlin/io/libp2p/pubsub/gossip/GossipScore.kt

```diff
@@ -117,6 +118,7 @@ class GossipScore(
         val ips = mutableSetOf<String>()
         var connectedTimeMillis: Long = 0
         var disconnectedTimeMillis: Long = 0
+        var cachedScore: AtomicReference<Double> = AtomicReference(0.0)
```

Comment:
```suggestion
        val cachedScore: AtomicReference<Double> = AtomicReference(0.0)
```

Reply:
Yes, you're right, cachedScore is a val.